test/notebook_tests/test_notebooks.py

```diff
@@ -83,6 +85,7 @@ def test_ipynb(dir_path, notebook, s3_bucket, region):
                 notebook, [errors[row]["evalue"] for row in range(len(errors))]
             )
         else:
+            pytest.skip()
```

Comment:
It skipped the notebook testing, but marked it as passed.

Reply:
The environment.yml file will enable us to test notebooks locally using the same set of dependencies as NBIs. So you just have run the command that is mentioned in the README and then you activate the environment and then can either run tests or develop the notebook without worrying whether this will work on NBIs or not. 
Also, because conda-pack upgrades most packages and is used in NBIs as well, the security issues caused by package versions will be reduced to some extent. This will save our time from manually checking package versions in NBIs and then upgrading them with dependabot.